docs/syntax/zenuml.md

```diff
@@ -376,15 +251,6 @@ zenuml
     }
 ```
 
-```mermaid
-zenuml
-    Alice->Bob: Hello Bob, how are you?
-    Bob->Alice: Not so good :(
-    opt {
-      Bob->Alice: Thanks for asking
-    }
-```
-
```

Comment:
The mermaid code is present for github to render the diagram, and the mermaid-example is present to show the code. Please don't remove the mermaid blocks.

Reply:
I intended to only change the mermaid-example to mmd. 
but the highliht is still not working, I changed this to draft for now.

Reply:
openned an issue to solve the problem. https://github.com/shikijs/textmate-grammars-themes/issues/93